libs/utils/utils.js

```diff
@@ -452,6 +455,8 @@ export async function loadBlock(block) {
   const base = miloLibs && MILO_BLOCKS.includes(name) ? miloLibs : codeRoot;
   let path = `${base}/blocks/${name}`;
 
+  MILO_BLOCKS_PRELOADS[name]?.forEach((preload) => loadLink(`${base}${preload}`, { rel: 'preload', as: 'script', crossorigin: 'anonymous' }));
```

Comment:
If a block is used in the preload array, then the block.css will still be awaited and will negate the perf gain.  We should check for js vs css and set the `as` param accordingly

Reply:
The current solution is to preload imports in a block. decorate.js is always imported by marquee. It's not dealing with CSS.
```
const MILO_BLOCKS_PRELOADS = { marquee: ['/utils/decorate.js'] };
```

Reply:
It is not dealing with CSS right now, but if we're looking for a long term solution we should consider this as well.